src/helpers/checkPermissions.js

```diff
@@ -0,0 +1,15 @@
+// import db from '../models';
+
+// const { permissions } = db;
+
+// export default () => {
+//   const finalarray = [];
+//   permissions.articles.forEach(e1 => permissions.normal.forEach((e2) => {
+//     if (e1 === e2) {
+//       finalarray.push(e1);
+//     }
+//   }));
+//   return finalarray;
+// };
+
+// userPermissions.articles.includes('delete');
```

Comment:
You commented all lines in this file

Reply:
This file no longer exist, we now have a middleware for checking permissions.